eox_tenant/models.py

```diff
@@ -194,7 +194,7 @@ def get_configs_for_domain(cls, domain):
         config = TenantConfig.objects.get_configurations(domain=domain)
 
         if config:
-            return config["lms_configs"], config["external_key"]
+            return config[config_key], config["external_key"]
```

Comment:
@navinkarkera Would it make sense to add a new field called something like `common_configs` which would hold configs applicable for both lms and studio ? These values could then be merged with `config_key` values before returning.

Since in the existing implementation `lms_configs` field was used for both the LMS and studio, perhaps there might be studio related settings configured in instances using this repo. To prevent these instances from breaking, we could introduce a migration to move data from `lms_configs` to the new common field.

Reply:
@kaustavb12 I had the same thought but dropped the idea temporarily to avoid making a lot of changes in the codebase. For now I have updated the tutor-contrib-grove plugin to add required base settings to both the fields by default.

Reply:
@navinkarkera  Yes, this would not affect us that much, but I am worried about upstream acceptability. Perhaps we can consult the upstream reviewer before moving forward with this.